test/index.test.js

```diff
@@ -229,6 +229,66 @@ describe('Validator Object tests', () => {
         )
 
     })
+
+    it("Should return true when isValid is invoked on object having no errors", () => {
+        const constraints = {
+            person: {
+                children: [
+                    {
+                        name: "nonEmptyString",
+                        age: "NumberStrategy:positiveInteger"
+                    }
+                ]
+            }
+        };
+
+        const validator = new Maanyakaran(constraints);
+
+        let outcome = validator.validate({
+            person:
+                {
+                    children: [
+                        {
+                            name: "peter",
+                            age: 1
+                        }
+                    ]
+                }
+        })
+        expect(outcome.isValid()).toBe(true);
+        expect(outcome).toEqual({person: {children: {"0": null}}});
```

Comment:
Can we have test cases for multiple keys with valid and invalid states and check flag for each key and its parent.